.github/workflows/CI-DOCKER-DPDK.yml

```diff
@@ -0,0 +1,52 @@
+name: "p4c-dpdk-ptf-p4testgen-tests"
```

Comment:
Can you merge this with the existing file?

Reply:
sure